src/core/noelle/src/Noelle_loops.cpp

```diff
@@ -267,24 +262,9 @@ LoopDependenceInfo *Noelle::getLoop(
   /*
    * Fetch the loop index.
    */
-  auto loopIndex = this->loopHeaderToLoopIndexMap.at(header);
-
-  /*
-   * No filter file was provided. Construct LDI without profiler configurables
-   */
-  if (!this->hasReadFilterFile) {
-    auto ldi =
-        this->getLoopDependenceInfoForLoop(header,
-                                           funcPDG,
-                                           DS,
-                                           0,
-                                           8,
-                                           this->om->getMaximumNumberOfCores(),
-                                           optimizations);
-
-    delete DS;
-    return ldi;
-  }
+  auto loopIDOpt = loop->getID();
```

Comment:
Same comment about requiring having an ID for all loops

Reply:
same yes